1.data-exploration/4.cell_line_summary_figure.ipynb

```diff
@@ -42,7 +42,7 @@
    "outputs": [],
    "source": [
     "# Set i/o paths and files\n",
-    "data_dir <- file.path(\"../0.data-download/data\")\n",
+    "data_dir <- file.path(\"0.data-download/data\")\n",
```

Comment:
Why did this path change? I'm not sure if this will work now, if we are using relative paths